test/puppetlabs/puppetdb/integration/smoke.clj

```diff
@@ -27,3 +28,94 @@
           (is (= catalog-uuid report-uuid)))))))
 
 
```

Comment:
Is the link to the friendly format doc (examples, etc.) "stable"?  If so, let's put it in a comment here.

Reply:
I'm hesistant to link to the doc because it's inaccurate, but I can create a new one that's accurate and link it here if you want

Reply:
Oh, I meant *their* doc.  If puppet doesn't have an official one yet for this format, then we can wait, but we should eventually have something to refer to.